StartBonusActivation/handler.ts

```diff
@@ -116,6 +118,14 @@ export function StartBonusActivationHandler(
           )
       )
       .fold(identity, apiBonusActivation => {
+        // Add the tuple (fiscalCode, bonusId) to the processing bonus collection
+        // this is used a s lock to avoid having more than one bonus in processing status
+        // The lock is removed when the orchestrator terminate
+        // tslint:disable-next-line: no-object-mutation
+        context.bindings.processingBonusIdOut = ProcessingBonus.encode({
```

Comment:
here we write the lock (it's probably better to do this at the beginnig using the sdk; i'm gonna refactor this)